CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,6 @@
+# Code of Conduct
```

Comment:
We also have this in the main readme: https://github.com/dotnet/command-line-api#code-of-conduct.

Any changes warranted there?

Reply:
The content is OK, but I'd probably remove it. FWIW, our policy tooling will flag [referencing the wrong CoC](https://github.com/dotnet/org-policy/blob/master/doc/PR16.md).